.github/workflows/release.yml

```diff
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - release-*
+    tags: [ v.* ]
```

Comment:
There is even a trigger on GitHub release. But this is what we did in Travis.

Reply:
Will this trigger for regular pushes to those branches so that it builds snapshots? Or does it build only if there is a v-tag?

Reply:
It will build for tags and the branches defined.

Reply:
I haven't actually tested the tag build though.

Reply:
I'll keep the trigger on tag for now.